### PR TITLE
TS-1: Committing changes to models.py for BasicClassSession and Instructor …

### PR DESCRIPTION
**Pull request by jleodones**
…models + aux models.

### CODE REVIEW

**Comment by jleodones** [comment]
[TS-2: Create models in Classes](https://trello.com/c/632queDa/2-ts-2-create-models-in-classes)

File: apps/classes/models.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from curses.ascii import NUL
```
Comment (markkyaw) [inline apps/classes/models.py]: What is this for?

File: apps/classes/models.py [hunk 2]
```diff
@@ -38,3 +53,43 @@ class BasicClassInfo(models.Model):
...
+    - Type - done
+    - Time
+        - Start time
+        - End time
+    - Days - done
+    - Instructor - done
+    - Location
+        - Building
+        - Room #
+
```
Comment (markkyaw) [inline apps/classes/models.py]: Can you provide example of what it'll look like
ex
```
session: 12312R
Session - 001
Type - 1
... etc
```

File: apps/classes/models.py [hunk 3]
```diff
@@ -38,3 +53,43 @@ class BasicClassInfo(models.Model):
...
+    Notes:
+    - Maybe implement default values for future semesters when not all values have been assigned.
+    """
+
```
Comment (markkyaw) [inline apps/classes/models.py:76]: `section` missing

File: apps/classes/models.py [hunk 4]
```diff
@@ -38,3 +53,43 @@ class BasicClassInfo(models.Model):
...
+    session = models.CharField(
+        max_length=3,
+    )
+    type = models.IntegerField(choices=SectionType.choices)
```
Comment (markkyaw) [inline apps/classes/models.py]: Rename to `section_type` since `type()` is a function in python for better clarity

File: apps/classes/models.py [hunk 5]
```diff
@@ -9,7 +11,20 @@ class ClassSemester(models.IntegerChoices):
...
+    LECTURE = 1, _("Lecture")
+    DISCUSSION = 2, _("Discussion")
+    LAB = 3, _("Lab")
+    QUIZ = 4, _("Quiz")
+
+
 # Create your models here.
+class Instructor(models.Model):
+    name = models.CharField(
+        max_length=200,
```
Comment (markkyaw) [inline apps/classes/models.py:22]: Look into `related_name`
https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.ForeignKey.related_name

**Review by markkyaw — CHANGES_REQUESTED** [review]

File: apps/classes/models.py (hunk 1 above)
Comment (jleodones) [reply apps/classes/models.py]: u wrote it not me homie

File: apps/classes/models.py (hunk 4 above)
Comment (jleodones) [reply apps/classes/models.py]: done

File: apps/classes/models.py (hunk 3 above)
Comment (jleodones) [reply apps/classes/models.py:76]: isnt this inherited automatically?

File: apps/classes/models.py (hunk 1 above)
Comment (markkyaw) [reply apps/classes/models.py]: How do I write if it's in your branch LOL. It's probably from stray import. You can remove it

Comment (jleodones) [reply apps/classes/models.py]: IDFK

**Review by markkyaw — APPROVED** [review]

**Pull request merged**